### PR TITLE
RIP-0015 base 2.5 proposal

### PR DESCRIPTION
**Pull request by Yuripetusko**
I am proposing here a BASE structure that saves type-specific props offchain on IPFS and accepts arbitrary fields utilising metadata properties structure

### CODE REVIEW

File: standards/rmrk2.0.0/entities/base.md [hunk 1]
```diff
@@ -163,6 +163,17 @@ needs, like so:
...
 
+
+### Mixed Base type
+
+The mixed Base is a simple Base type, that is similar to SVG Base, but a renderer should not assume that every part is SVG and instead check the mime type of each part separately to decide how to render it.
+
+The parts will have Z indexes for layered rendering, and the final NFT is a composited. The Mixed type implies the following:
+
+- every part will be absolutely positioned on top of each other, based on `z` field
+- every part will be positioned in the same viewport at coordinates 0,0 unless optional `x` and `y` fields are not provided
```
Comment (Swader) [inline standards/rmrk2.0.0/entities/base.md:174]: are provided

File: standards/rmrk2.0.0/entities/base.md [hunk 2]
```diff
@@ -163,6 +163,17 @@ needs, like so:
...
+- if a part's `src` file is of mime type of video or audio and this part has a `thumb` field, then the `thumb` is displayed to the user, and `src` file is played on click, however if `thumb` is absent, then a standard audio or video player is rendered
```
Comment (Swader) [inline standards/rmrk2.0.0/entities/base.md:175]: played on click of what?

Comment (Swader) [reply standards/rmrk2.0.0/entities/base.md:175]: And where is thumb displayed? Layered?

Comment (Yuripetusko) [reply standards/rmrk2.0.0/entities/base.md:175]: Actually this secion is not needed, it's deprecated after the new base proposal below

**Pull request merged**